tests/uefistored/test_cert_inheritance.py

```diff
@@ -6,17 +6,19 @@
 
 from utils import generate_keys, revert_vm_state
 
+CERT_DIR = "/var/lib/uefistored"
```

Comment:
There should be a "requirements" comment here (which should document the requirement of at least 2 hosts in the pool for TestPoolToDiskCertInheritance).